packages/btp-utils/src/destination.ts

```diff
@@ -224,3 +227,16 @@ export function isS4HC(destination: Destination): boolean {
             destination.ProxyType === ProxyType.INTERNET
     );
 }
+
+/**
+ * Checks if the destination attributes WebIDEUsage is configured with odata_abap.
+ *
+ * @param destination destination configuration properties
+ * @returns true, if this destination has the the 'odata_abap' attribute set
+ */
+export function isAbapODataDestination(destination: Destination): boolean {
+    return (
```

Comment:
The majority of SAP BTP destinations created, have both `odata_gen` and `odata_abap` specified which means this will always be false! I think the as-is flow needs to remain which is if `odata_abap` is found first, that determines if it's an ABAP destination?

Reply:
It cant be both though so that would be a misconfigured destination, which we shouldnt support?

Reply:
OK for now will revert...thx

Reply:
Thanks Iain.